example-wasm/src/setupTests.ts

```diff
@@ -0,0 +1,5 @@
+// jest-dom adds custom jest matchers for asserting on DOM nodes.
+// allows you to do things like:
+// expect(element).toHaveTextContent(/react/i)
+// learn more: https://github.com/testing-library/jest-dom
+import '@testing-library/jest-dom';
```

Comment:
Do we need this?  If so, maybe some rudimentary tests should be written.

Reply:
Haven't included any React tests but I left those default files untouched. Similar to the `reportWebVitals`, not entirely sure if it should stay or be removed from the example?

Reply:
I suppose whatever is generated as part of CRA could stay.  Was just unsure what the intention was.